.gitignore

```diff
@@ -1,5 +1,2 @@
-test-db/
-test-db-*/
 coverage
-node_modules/
-*.log
```

Comment:
got it, yes my `~/.gitignore` file looks like this

```
.DS_Store
npm-debug.log
```

But okay, we can put this into our repos, too, I don't mind :) I'll make an issue so we don't forget